scst-store/logs.hbs.md

```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
 * Some information in these payloads may be sensitive, and the user may not want them exposed in
 production environment logs.
 
+##  <a id='graphql-endptlog-out'></a> GraphQL endpoint log output (Only applicable to Artifactory Metadata Repository logs)
+
+When an GraphQL endpoint handles a request, the AMR generates two and five log lines. They are:
+
+1. Every request will have received by endpoint will have `Processing request`. This will include operation called and actual fields asked in the response output 
+1. Every response will yield actual query and status
+1. If the endpoint returns a response body, it outputs a second `Request response` line with an extra
+key `payload`, and its value is set to the entire response body. This line is shown at the `debug`
+   verbosity level.
```

Comment:
we should either have all end with a period or with none. (im not too familiar what the style guide is)